pkg/lib/queueinformer/resourcequeue.go

```diff
@@ -68,6 +69,29 @@ func (r *ResourceQueueSet) Requeue(namespace, name string) error {
 	return fmt.Errorf("couldn't find queue for resource")
 }
 
+// TODO: this may not actually be required if the requeue is done on the namespace rather than the installplan
+// RequeueAfter requeues the resource in the set with the given name and namespace (just like Requeue), but only does so after duration has passed
+func (r *ResourceQueueSet) RequeueAfter(namespace, name string, duration time.Duration) error {
```

Comment:
I don't see this used anywhere?

Reply:
This is being used in the [installplan sync](https://github.com/njhale/operator-lifecycle-manager/blob/unpack-bundles/pkg/controller/operators/catalog/operator.go#L1139).